crates/red_knot_python_semantic/src/types/display.rs

```diff
@@ -183,21 +183,21 @@ impl Display for DisplayLiteralGroup<'_> {
 enum LiteralTypeKind {
     Class,
     Function,
-    IntLiteral,
-    StringLiteral,
-    BytesLiteral,
+    Int,
+    String,
+    Bytes,
```

Comment:
Instead of adding suffixes to the `Class` and `Function` variants, I removed the suffix from the other remaining variants.

Reply:
Makes sense. I think clippy would also start yelling at you if all have the same suffix :)

Reply:
Exactly (see PR description)